browsers/browsers.json

```diff
@@ -1422,5 +1422,65 @@
         "status": "current"
       }
     }
+  },
+  "samsunginternet_android": {
```

Comment:
Overall, LGTM.

Are there release notes available on the Web, that we could link from here?

Reply:
Thanks @teoli2003. There are some brief release notes here, but it is only one URL that keeps getting its content updated, rather than individual URLs for each version: http://developer.samsung.com/internet/android/releases Would this still be useful to include? NB. I can ask about having separate pages for each browser version, perhaps we can do that in future.

Reply:
We can adjust the release dates and the release notes URLs in future PRs, if necessary.
I'm planning to display the release date as a tooltip when hovering over release versions in the redesigned MDN compat table and we might link to release notes from there, too. That's not yet implemented, though.